recipe/meta.yaml

```diff
@@ -9,12 +9,12 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 742645059757a56ecd886faf4ed2441b9c0cd406079c2b4bee51bcc3fbcd510a
+  sha256: 91b8e218852ef6007c2b98cd861601c6a09f1aa32bbbb74fab5b1c33d4a1e410
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
-  skip: true  # [py<36]
+  skip: true  # [py<=36]
 
 requirements:
   build:
```

Comment:
I think `cython >=0.29.32` in `host` will be better. See https://github.com/pydantic/pydantic/blame/v1.10.2/requirements.txt#L3

Reply:
Okay, should I also add `python-dotenv>=0.10.4` to the run-constrained as well?
See https://github.com/pydantic/pydantic/blob/v1.10.2/setup.py#L137

Reply:
`python-dotenv` is not available on our default channel `main` https://anaconda.org/search?q=python-dotenv

Reply:
Oh, that's a great resource for me to check for future reference. Thanks so much!

Reply:
You can also use `conda search` if you want to check against those matchspecs:
```bash
conda search --subdir linux-64 'python-dotenv>=0.10.4'
```
You can change that subdir to make sure a package exists on each of the platforms you need to build on.

Reply:
Oh nice! Thanks!